### ISSUE TITLE
Read JSON data from Python/Fortran

### ISSUE THREAD

**Issue by edigiacomo** [ARPA-SIMC/dballe#186]
It would be useful to read a JSON file from Python:

```python
importer = dballe.Importer("JSON")
with importer.from_file("test.json") as f:
    for msg in f:
        print("{m.report},{m.coords},{m.ident},{m.datetime},{m.type}".format(m=msg))
```

**Comment by spanezz** [comment]
It should work now, after the fix of #202

**Comment by edigiacomo** [comment]
I confirm that it works (with a little delay :grimacing: )
